### ISSUE TITLE
Error during load

### ISSUE THREAD

**Issue by aemaddin** [dcasia/nova-filepond#3]
Laravel 6
Nova 2

after uploading image, can't load the uploaded image from storage.

Post resource code :

```
 public function fields(Request $request)
    {
        return [
            ID::make()->sortable(),

            Filepond::make('image')
                ->disk('public', 'posts')
                ->rules('required'),

            Text::make('title')
                ->rules('required'),

            Trix::make('description'),
        ];
    }
```

error screenshot

<img width="1134" alt="Screen Shot 2019-10-06 at 2 05 46 PM" src="https://user-images.githubusercontent.com/11630742/66268944-cad5b700-e842-11e9-8a74-a48419bdd5ab.png">

Laravel error log

[laravel-2019-10-06.log](https://github.com/dcasia/nova-filepond/files/3694320/laravel-2019-10-06.log)

**Comment by milewski** [comment]
That's due the second argument of disk method `->disk('public', "PATH")`, let me fix it...

**Comment by milewski** [comment]
Fixed on version `v0.2.3`
